### ISSUE TITLE
Tie cache file name to `reentry` executable location instead of python

### ISSUE THREAD

**Issue by DropD** [DropD/reentry#25]
## Problem:
Typically a python env's `bin/` contains links like
```
bin/
  python  --> python2
  python2.7 --> python2
  python2
  etc
```
Currently depending on which one of them you use to run your code, you will get a different `reentry` entry point cache. The link name can not be resolved because the python executable in multiple environments might be just links to one and the same binary.

## Proposed solution:
Create a `reentry dev` subcommand that only tells it's location. console_scripts get installed in the virtual environment's bin folder even for `pip -e` installs.

**Comment by DropD** [comment]
The proposed solution will not work when `reentry` is installed as a build time dependency. The `reentry` console script gets installed along with `reentry` itself into a temporary build environment. The same probably goes for asking distutils for the library path (not tested yet).

Nevertheless the problem has ben solved for now by using the directory in which the python executable resides and appending a string to differenciate between py2 and py3.

I will close this and open another issue to think about a more elegant solution.

**Comment by DropD** [comment]
fixed in v1.2.1-alpha.2
